### PR TITLE
Creating database and connecting with models

### PR DESCRIPTION
**Pull request by temp-gt**
Please review following changes.

### CODE REVIEW

File: blog/main.py [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
+from .database import engine
+
+models.Base.metadata.create_all(bind=engine)
 
 app = FastAPI()
 
 
 @app.post('/blog')
 def create_blog(request: Blog):
-    blogs.BLOGS.append(request)
```
Comment (shahid-gt) [inline blog/main.py:10]: Please make sure to support this on later version.

File: blog/models.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+from .database import Base
+from sqlalchemy import Column, Integer, String, Boolean
+
+
+class Blog(Base):
+    __tablename__ = 'blogs'
```
Comment (shahid-gt) [inline blog/models.py:6]: Add line below.

**Review by shahid-gt — CHANGES_REQUESTED** [review]
Looks good to me.

**Review by shahid-gt — CHANGES_REQUESTED** [review]
w

File: blog/main.py (hunk 1 above)
Comment (temp-gt) [reply blog/main.py:10]: Done

File: blog/models.py (hunk 2 above)
Comment (temp-gt) [reply blog/models.py:6]: Done

**Review by temp-gt — COMMENTED** [review]
d

**Review by shahid-gt — APPROVED** [review]
LGTM

**Review by temp-gt — COMMENTED** [review]
LGTM

**Review by shahid-gt — COMMENTED** [review]
LGTM

**Pull request merged**